compiler/luci/pass/src/QuantizationUtils.h

```diff
@@ -63,6 +64,9 @@ void propagate_pad_v2_quantparam(luci::CirclePadV2 *pad_v2);
 // Return true if the node is quantized
 bool is_quantized(const CircleNode *node);
 
+// Cast a floating-point number to an unsigned 8-bit integer
+uint8_t fp32_to_uint8_cast(float f);
```

Comment:
why add this function prototype without any implementation ?
I can't find any caller in draft.

Reply:
You can find them on lines +325 to +332 in [QuantizeWeightsWithGPTQPass.cpp](https://github.com/Samsung/ONE/pull/13585/files#diff-6088c0fd93930b0e8c034c0dc0e69fc1ba883a3bacd012d1e548c63731be8ec9)